lib/Doctrine/DBAL/Migrations/Provider/LazySchemaDiffProvider.php

```diff
@@ -39,7 +39,7 @@ public function __construct(LazyLoadingValueHolderFactory $proxyFactory, SchemaD
         $this->originalSchemaManipulator = $originalSchemaManipulator;
     }
 
-    public static function fromDefaultProxyFacyoryConfiguration(SchemaDiffProviderInterface $originalSchemaManipulator)
+    public static function fromDefaultProxyFactoryConfiguration(SchemaDiffProviderInterface $originalSchemaManipulator)
```

Comment:
it actually is. Preserving Bc would be easy, so I suggest doing it: create a method with the old name which would call the new method (and trigger a deprecation notice)